### ISSUE TITLE
CustomerClientServiceClient is not in v14

### ISSUE THREAD

**Issue by makstech** [googleads/google-ads-php#747]
**Your client library and Google Ads API versions:**

* Client library version: `v14.0.0`
* Google Ads API version: `V10`

**Description of the bug:**

Since at least Google Ads API V2 up to V9, there was a [`\Google\Ads\GoogleAds\V9\Services\CustomerClientServiceClient`](https://github.com/googleads/google-ads-php/blob/v13.0.0/src/Google/Ads/GoogleAds/V9/Services/CustomerClientServiceClient.php). Now, with library v14 (API v10), it's not there anymore. No mentions of these changes can be found in this project nor in the API migration guide.

**Comment by PierrickVoulet** [comment]
Hi @MaksTech,

It is an expected change because `get` methods have been removed for all services. Because `CustomerClientService` exclusively had a `get` method it has been removed as well. This change is discussed in the [migration section](https://developers.google.com/google-ads/api/docs/version-migration#v9-v10) of the Google Ads API documentation. You are right that it was not stated in the [release notes](https://developers.google.com/google-ads/api/docs/release-notes#v10) though, I will provide the feedback to our teams to fix this.

Does this change break your implementation in any way (from a library perspective)? If it does, could you give more details?

**Comment by MaksTech** [comment]
Hi, 

Thanks for the explanation!  

I was primarily using it for the `CustomerClientServiceClient::parseName()` method to get the customer IDs after performing search on the [`CustomerClient`](https://developers.google.com/google-ads/api/reference/rpc/v10/CustomerClient) resource (`customer_client.resource_name`).

**Comment by PierrickVoulet** [comment]
Got it. We opened a feature request https://github.com/googleapis/gapic-generator-php/issues/453 to improve.

**Comment by chiragvels** [comment]
Hi @PierrickVoulet ,

I would also like to add this parseName functionality.

I was using same for click_view.
To get the  user_list_id and criterion_id as it is not directly available.

```
UserListServiceClient::parseName()
AdGroupCriterionServiceClient::parseName()
```

**Comment by fiboknacky** [comment]
https://github.com/googleapis/gapic-generator-php/issues/453 is already marked closed, so I'm closing this.
Now, the classes are generated with `parseName()` functions available in some key classes since [v19.2.0](https://github.com/googleads/google-ads-php/releases/tag/v19.2.0)
Overall, they may not be exactly the same as before, but it should reflect more closely to the proto file structure now.
